server/models/handlers/availability.js

```diff
@@ -1,29 +1,22 @@
 // Gets the time availabilities for a specific worker for a specific school.
-async function getWorkerAvailability(workerId, schoolId, startTime, endTime) {
-    const sampleAvailableTimes = [
-        {
-            workerTimeslotId: 1,
-            startTime: '08:00:00', 
-            endTime: '08:30:00',
-            date: '2020-10-20'
-        },
-        {
-            workerTimeslotId: 2,
-            startTime: '08:30:00', 
-            endTime: '09:00:00',
-            date: '2020-10-20'
-        },
-        {
-            workerTimeslotId: 3,
-            startTime: '09:00:00', 
-            endTime: '09:30:00',
-            date: '2020-10-20'
-        }
-    ];
+const _ = require('lodash');
+const availabilityModel = require('../db/availability');
+//need userTypes or Workerstatus?
```

Comment:
Hi Nancy,
I would recommend adding more details to this comment OR removing it as it may confuse future developers.

Reply:
Hi Amy,
Good catch!
My action: I've removed the comment on line 4 of the server/models/handlers/availability.js

Please confirm if this takes care of your suggestion!